composer/models/tasks/classification.py

```diff
@@ -49,34 +51,44 @@ class ComposerClassifier(ComposerModel):
         from composer.models import ComposerClassifier
 
         pytorch_model = torchvision.models.resnet18(pretrained=False)
-        model = ComposerClassifier(pytorch_model)
+        model = ComposerClassifier(pytorch_model, num_classes=1000)
     """
 
     num_classes: Optional[int] = None
 
     def __init__(self,
                  module: torch.nn.Module,
+                 num_classes: Optional[int] = None,
                  train_metrics: Optional[Union[Metric, MetricCollection]] = None,
                  val_metrics: Optional[Union[Metric, MetricCollection]] = None,
                  loss_fn: Callable = soft_cross_entropy) -> None:
         super().__init__()
 
+        self.module = module
+        self._loss_fn = loss_fn
+
+        self.num_classes = num_classes
+        if hasattr(self.module, 'num_classes'):
+            self.num_classes = getattr(self.module, 'num_classes')
+        if self.num_classes == None and (train_metrics is None or val_metrics is None):
+            raise ValueError(
+                textwrap.dedent('Please specify the number of output classes. Either: \n (1) pass '
+                                'in num_classes to the ComposerClassifier \n (2) pass in both '
+                                'train_metrics and val_metrics to Composer Classifier, or \n (3) '
+                                'specify a num_classes parameter in the PyTorch network module.'))
+
         # Metrics for training
         if train_metrics is None:
-            train_metrics = Accuracy()
+            assert self.num_classes is not None
+            train_metrics = MulticlassAccuracy(num_classes=self.num_classes)
```

Comment:
just to double check, does `MulticlassAccuracy` just become `BinaryAccuracy` if you pass `num_classes=2`?